src/domain/UBGraphicsScene.h

```diff
@@ -415,6 +417,8 @@ public slots:
         void hideMarkerCircle();
         void hidePenCircle();
         void DisposeMagnifierQWidgets();
+        void MultiTouchDrawing(QTouchEvent* event, UBStylusTool::Enum currentTool);
+        void MultiTouchEndDrawing();
```

Comment:
I most cases (well, most, there is a counter-example in the previous line, Sigh!) we start a method name with a lower case letter.